app/models/container_image.rb

```diff
@@ -24,6 +24,7 @@ class ContainerImage < ApplicationRecord
   has_many :openscap_rule_results, :through => :openscap_result
   has_many :labels, -> { where(:section => "labels") }, :class_name => CustomAttribute, :as => :resource, :dependent => :destroy
   has_many :docker_labels, -> { where(:section => "docker_labels") }, :class_name => CustomAttribute, :as => :resource, :dependent => :destroy
+  has_many :openshift_annotations, -> { where(:section => "openshift_annotations") }, :class_name => CustomAttribute, :as => :resource, :dependent => :destroy
```

Comment:
might be missing something here...
why are we naming these `openshift_annotations` and no annotations?
(we might have in the future openshift_annotation and kubernetes_annotations in different entities based on repository, but they are the same thing)